### DISCUSSION TITLE
is centreon-api down?

### DISCUSSION THREAD

**Discussion by yosiasz** [centreon-archive/centreon-archived#11212] [General]
Greetings,

Is this no longer available?

https://raw.githubusercontent.com/centreon/centreon/develop/doc/API/centreon-api-v22.04.yaml

Thanks!

**Comment by ponchoh** [comment]
https://github.com/centreon/centreon/blob/22.04.x/doc/API/centreon-api-v22.04.yaml

## RAW

https://raw.githubusercontent.com/centreon/centreon/22.04.x/doc/API/centreon-api-v22.04.yaml

**Reply by yosiasz** [reply]
thanks @ponchoh

**Reply by yosiasz** [reply]
https://docs.centreon.com/docs/api/rest-api-v2/ still referring to the old one in case someone needs to fix that up.

**Reply by ponchoh** [reply]
TY 👍 
https://github.com/centreon/centreon-documentation/pull/1599
😄
